src/main/java/com/softserve/actent/controller/EventController.java

```diff
@@ -0,0 +1,36 @@
+package com.softserve.actent.controller;
+
+import com.softserve.actent.model.dto.converter.EventConverter;
+import com.softserve.actent.model.dto.event.EventCreationDto;
+import com.softserve.actent.model.entity.Event;
+import com.softserve.actent.service.EventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1")
+public class EventController {
+
+    private final EventService eventService;
+    private final EventConverter eventConverter;
+
+    @Autowired
+    public EventController(EventService eventService, EventConverter eventConverter) {
+        this.eventService = eventService;
+        this.eventConverter = eventConverter;
+    }
+
+    @PostMapping(value = "/events")
+    public ResponseEntity<Long> addEvent(@RequestBody EventCreationDto eventCreationDto) {
```

Comment:
Implement your controller methods without ResponseEntity, check out this class: https://github.com/lv-381-java/Actent/blob/develop/src/main/java/com/softserve/actent/controller/TagController.java

Reply:
Also now we are using IdDto as we want JSON response, but not just pure Long/String. Maybe there is any other solution, but until this moment we were stuck with this approach.

Reply:
Approved by accident, in fact am asking for changes.

Reply:
Only after team decision or after Ihor approve it!

Reply:
Whole team used IdDto's everywhere, it counts as team decision? Also I discussed it with Viktoriya and she 
acknowledged this as maybe not so good, but solution.

And what's about ResponseEntity - Ihor mentioned this twice already in various PRs.

Reply:
ok